test/BaselineOutput/Common/LightGBMMC/LightGBMMC-CV-iris.key-out.txt

```diff
@@ -23,6 +23,10 @@ Accuracy(micro-avg): 0.936709
 Accuracy(macro-avg): 0.942857
 Log-loss:           0.285741
 Log-loss reduction: 0.737254
+Top K accuracy(All K)K=1: 0.936709
```

Comment:
Does anyone know where in the code is it decided to print this information and not, for example,  `PerClassLogLoss`,  or `TopKAccuracy` ?

Reply:
It's the parameters. `outputPerClass` and `outputTopKAcc` are both falsy. In the case of TopK, obviously that's because we're now ignoring it. Also, note this is in the MAMLEvalutator. A *lot* of the tests exercise the MAML evaluator.

https://github.com/dotnet/machinelearning/blob/b7ded43a5aa24057cae4dbede258ed39c757539a/src/Microsoft.ML.Data/Evaluators/MulticlassClassificationEvaluator.cs#L912-L917

Reply:
MAML is publicly deprecated but we do use it inside the repository to run many of our tests. Still, it's weird that in these outputs we get the top-k values, despite the fact that in your PR you actually drop the Top-k column on the MamlEvaluator, so i think i'm missing something

Reply:
The above code snippet is for the `PrintFoldResultsCore` implementation. That includes pretty much all metrics. 

On the other hand, `DropAllTopKColumn` is only called by `CombineOverallMetricsCore` which is essentially preparing data for `EvalutatorUtils.ConcatenateOverallMetrics` which is what doesn't like the per-class and per k metrics.

`TrainTestCommand.RunCore()` calls both `evaluator.PrintFoldResults(ch, metrics)` and `evaluator.GetOverallResults(overall)` to end up executing both those paths.

Reply:
I've changed it so that both branches output the All top K, but since they now respect the parameter, they don't output anything on most of the tests.